### ISSUE TITLE
Clean out old large files

### ISSUE THREAD

**Issue by seananderson** [vast-lib/tinyVAST#14]
- [x] .dll
- [x] .o
- [x] compiled PDFs (lower priority, not as big)

I can do it quickly with https://rtyley.github.io/bfg-repo-cleaner/ while maintaining git history.

But, this needs to be done before public access because everyone with a copy needs to reclone after to avoid pushing back old files.

For this to happen, I just need to know you, @James-Thorson-NOAA, have no local edits that haven't been committed and pushed. So, let me know when you have a clean local repo and I'll do this.
